### PR TITLE
[WIP] Dont display register form for cart recovery

### PR DESCRIPTION
**Pull request by PululuK**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Dont display register form for cart recovery.
| Type?             | new feature 
| Category?         | FO 
| BC breaks?        | no
| Deprecations?     |  no
| Fixed ticket?     | N/A
| How to test?      | N/A
| Possible impacts? | N/A

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25728)

### CODE REVIEW

File: themes/classic/templates/checkout/_partials/steps/personal-information.tpl [hunk 1]
```diff
@@ -48,22 +48,24 @@
...
-          {if !$show_login_form} aria-selected="true"{/if}
-          >
-          {if $guest_allowed}
-            {l s='Order as a guest' d='Shop.Theme.Checkout'}
-          {else}
-            {l s='Create an account' d='Shop.Theme.Customeraccount'}
-          {/if}
-        </a>
-      </li>
+      {if $show_register_form }
```
Comment (kpodemski) [inline themes/classic/templates/checkout/_partials/steps/personal-information.tpl]:
```suggestion
      {if $show_register_form}
```

File: classes/checkout/CheckoutPersonalInformationStep.php [hunk 2]
```diff
@@ -110,6 +110,7 @@ public function render(array $extraParams = [])
                 'register_form' => $this->registerForm->getProxy(),
                 'guest_allowed' => $this->getCheckoutSession()->isGuestAllowed(),
                 'empty_cart_on_logout' => !Configuration::get('PS_CART_FOLLOWING'),
+                'show_register_form' => !Tools::getValue('recover_cart')
```
Comment (kpodemski) [inline classes/checkout/CheckoutPersonalInformationStep.php]:
```suggestion
                'show_register_form' => !Tools::getValue('recover_cart'),
```

**Comment by kpodemski** [comment]
Nice. I think QA needs an issue for this one to properly test everything 👍

**Review by sowbiba — APPROVED** [review]
Looks good. Thanks @PululuK

**Review by matthieu-rolland — COMMENTED** [review]
Looks good 👍 

I can't approve without an issue validated by @PrestaShop/product-team though

**Comment by matks** [comment]
Hello @PululuK what do we do with this PR?

**Comment by matks** [comment]
Hi @PululuK ,

Since we had no news from you for almost 2 months, I will follow [our guidelines](https://devdocs.prestashop.com/1.7/project/maintainers-guide/reviewing-pull-requests/#stale-pull-requests) and I'll close this pull request as it seems stale. Naturally if you want to continue this PR, you can reopen 😉  .

Thanks!

**Pull request closed without merging**